lib/grub_utils.pm

```diff
@@ -47,11 +47,11 @@ sub grub_test {
         bug_workaround_bsc1005313() unless get_var("BOOT_TO_SNAPSHOT");
     }
     else {
+        # Slow down the command 'return' execution speed to give Hyper-V some response time
+        wait_still_screen 2 if (check_var('VIRSH_VMM_FAMILY', 'hyperv') && get_var('UEFI'));
```

Comment:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/16477#discussion_r1121176445 still applies here. `wait_still_screen` is effectively a "sleep *at least* 2s". Please reconsider to find out the *actual event* you need to wait for.